layouts/footer.ejs

```diff
@@ -1,8 +1,46 @@
+<footer class="page-footer">
+  <div class="page-footer__content">
+    <div class="page-footer__sections">
+      <section class="page-footer__section">
+        <header class="page-footer__section-title">Pulsar</header>
+        <ul class="page-footer__links">
+          <li><a href="/download">Downloads</a></li>
+          <li><a href="https://docs.pulsar-edit.dev/">Documentation</a></li>
+          <li><a href="https://web.pulsar-edit.dev/">Package repository</a></li>
+        </ul>
+      </section>
+
+      <section class="page-footer__section">
+        <header class="page-footer__section-title">Team</header>
+        <ul class="page-footer__links">
+          <li><a href="/blog">Blog</a></li>
```

Comment:
```suggestion
          <li><a href="https://blog.pulsar-edit.dev">Blog</a></li>
```

Get this to match our new blog destination.

Reply:
Cool. Will `/blog` redirect to the subdomain automatically? Want to make sure we don't break existing permalinks.

Reply:
Good call, that needs to be added to this site in our redirects directory

Reply:
Will we be able to do it on a bulk basis? For instance, a wildcard rewrite of 

```
https://pulsar-edit.dev/blog/20240902-savetheclocktower-modern-tree-sitter-part-7.html
```

to

```
http://blog.pulsar-edit.dev/posts/20240902-savetheclocktower-modern-tree-sitter-part-7/
```

?

Reply:
I'd have to do some research about a wildcard redirect, although I'd be fine setting up redirects as needed for what we've written so far, since automating it a single time for what exists now won't be too difficult.